report/4.Solution/ParserImplementation.tex

```diff
@@ -115,7 +118,83 @@ \subsection{Implementing a recursive decent parser}
 The method then checks whether the new current symbol is an equals-symbol - if that's the case the declaration is followed by an expression - which means it's an declaration with an assignment otherwise the expression is set to null. 
 The method then returns a Declaration, which is a subclass of AstNode.  
 
+A more complex parsing routine is when an expression is to be parsed, since an expression in HCL may be either a function-call or any kind of value.
```

Comment:
Change "a more complex parsing routine is when an expression is to be parsed" -> "It is slightly more complex to parse an expression".

(Maybe) Change "function-call or any kind of value" -> "function-call or any kind of evaluation to a value".